### PR TITLE
Revise getting started page

### PR DESCRIPTION
**Pull request by TakLee96**
This PR connects #80 and connects #89 and connects #110

### CODE REVIEW

File: content/js/index.md [hunk 1]
```diff
@@ -26,40 +28,121 @@ cd ..
...
+
+
+
+<br/><br/>
+<a name="intro-portal"></a>
+# Before you begin
+
+There are a few steps to complete and a few concepts to introduce.
+
+## Portal
```
Comment (david90) [inline content/js/index.md]: We call it "Developer Portal"

File: content/js/index.md [hunk 2]
```diff
@@ -26,40 +28,121 @@ cd ..
...
+```
+
+## Container
+
+Container is the uppermost layer of skygear. In practice,
+`import skygear from 'skygear'` will give you a container instance at variable
+skygear. In most case you will only need one instance of container.
+
+The first things you need to interact with container is setting `endPoint` and
+`apiKey`. If you don't know what that is, please read the [Getting Started](/js/guide)
```
Comment (david90) [inline content/js/index.md]: This should scroll to portal part

**Comment by david90** [comment]
The section "Before you begin" and the description "There are a few steps to complete and a few concepts to introduce." looks a bit vague to me.

Other than that, other looks fine to me.

File: content/js/index.md [hunk 3]
```diff
@@ -1,65 +1,142 @@
...
+
+
+
+
+
+
+
+
+
+<br/><br/>
```
Comment (rickmak) [inline content/js/index.md]: brbr?

Comment (rickmak) [inline content/js/index.md]: I think md files should contains content only, less style/layout is better.

File: content/js/index.md [hunk 4]
```diff
@@ -1,65 +1,142 @@
...
+if not please check your node version or report an issue):
 
-/* ... */
-
-var skygear = require('skygear');
+``` bash
+cd skygear
+npm test
+cd ..
 ```
```
Comment (rickmak) [inline content/js/index.md]: Why ask the user to run test...? If you want the bundle.js. I think we should encourage the use of npm release. or download the bundle.js at github: https://github.com/SkygearIO/skygear-SDK-JS/releases

File: content/js/index.md (hunk 3 above)
Comment (TakLee96) [inline content/js/index.md]: I put brbr before every title that is mentioned in the menu on the left to make the web page looks better and 10 lines to make the md file more readable. Maybe I shall remove all the brbr and the 10 lines?

File: content/js/index.md (hunk 4 above)
Comment (TakLee96) [inline content/js/index.md]: @rickmak Maybe directly give them the link to the `bundle.js` file would be a better idea? I'm still exploring how to integrate with react.js using webpack.

File: content/js/index.md (hunk 3 above)
Comment (rickmak) [inline content/js/index.md]: Yes, please. It can be done by css.

File: content/js/index.md (hunk 4 above)
Comment (rickmak) [inline content/js/index.md]: You will got a direct link after this issue is resolved. Let remove this npm test first.

**Pull request merged**